src/pages/guidelines/color/implementation.mdx

```diff
@@ -0,0 +1,546 @@
+| [Layer set tokens](/guidelines/color/implementation#layer-set-tokens)               | Explicit tokens used to manually map the layering model onto components. They come in sets based on their level in the layering model and are part of the theme package.                               |
+| [Contextual layer tokens](/guidelines/color/implementation#contextual-layer-tokens) | Abstracted tokens that automatically infer value based on the context a component inside the layering model. They are part of the component package and don't have an explicit value assigned to them. |
+
+<Row>
+<Column colLg={12}>
+
+![Layering model token comparison](images/color-implementation-compare.png)
+
+</Column>
+</Row>
+
+<Caption>
+  A visual spec using layer set tokens (left) versus contextual tokens (right).
+</Caption>
+
+<!---
+### Token list
+
+| Contextual tokens         | Layer set 01                 | Layer set 02                 | Layer set 03                 |
+| ------------------------- | ---------------------------- | ---------------------------- | ---------------------------- |
+| `$layer`                  | `$layer-01`                  | `$layer-02`                  | `$layer-03`                  |
+| `$layer-active`           | `$layer-active-01`           | `$layer-active-02`           | `$layer-active-03`           |
+| `$layer-hover`            | `$layer-hover--01`           | `$layer-hover-02`            | `$layer-hover-03`            |
+| `$layer-selected`         | `$layer-selected-01`         | `$layer-selected-02`         | `$layer-selected-03`         |
+| `$layer-selected-hover`   | `$layer-selected-hover-01`   | `$layer-selected-hover-02`   | `$layer-selected-hover-03`   |
+| `$layer-accent`           | `$layer-acccent-01`          | `$layer-accent-02`           | `$layer-accent-03`           |
+| `$layer-accent-active`    | `$layer-acccent-active-01`   | `$layer-accent-active-02`    | `$layer-accent-active-03`    |
+| `$layer-accent-hover`     | `$layer-acccent-hover-01`    | `$layer-accent-hover-02`     | `$layer-accent-hover-03`     |
+| `$field`                  | `$field-01`                  | `$field-02`                  | `$field-03`                  |
+| `$field-hover`            | `$field-hover-01`            | `$field-hover-02`            | `$field-hover-03`            |
+| `$border-subtle`          | `$border-subtle-01`          | `$border-subtle-02`          | `$border-subtle-03`          |
+| `$border-subtle-selected` | `$border-subtle-selected-01` | `$border-subtle-selected-02` | `$border-subtle-selected-03` |
+| `$border-strong`          | `$border-strong-01`          | `$border-strong-02`          | `$border-strong-03`          |
+--->
+
+<br />
+
+### Layer set tokens
+
+Layer tokens are sets of explicit tokens used to manually map the layering model
+onto components. They are part of the theme package and come in predefined sets
```

Comment:
I would mention what package they sit in as this can be confusing. For example, if someone is using the `@carbon/styles` or `@carbon/react` package this would be available to them. They may read this and assume they need o install the theme package too.

Reply:
I thought that was the big difference between the two (if im remembering previous conversations correctly)? So knowing which package they're in is irrelevant?

Reply:
Right, since packages can re-export items from another package it's not necessary to know which package they may originally be defined in. Instead, a developer needs to know how to get it from the package that they're using.

For most teams, the hope is to use `@carbon/react` directly and consume carbon through this one package.